x-pack/plugins/snapshot_restore/public/application/sections/home/snapshot_list/snapshot_list.tsx

```diff
@@ -290,41 +257,41 @@ export const SnapshotList: React.FunctionComponent<RouteComponentProps<MatchPara
       />
     );
   } else {
-    const repositoryErrorsWarning = Object.keys(errors).length ? (
-      <Fragment>
-        <EuiCallOut
-          title={
-            <FormattedMessage
-              id="xpack.snapshotRestore.repositoryWarningTitle"
-              defaultMessage="Some repositories contain errors"
-            />
-          }
-          color="warning"
-          iconType="alert"
-        >
-          <FormattedMessage
-            id="xpack.snapshotRestore.repositoryWarningDescription"
-            defaultMessage="Snapshots might load slowly. Go to {repositoryLink} to fix the errors."
-            values={{
-              repositoryLink: (
-                <EuiLink {...reactRouterNavigate(history, linkToRepositories())}>
-                  <FormattedMessage
-                    id="xpack.snapshotRestore.repositoryWarningLinkText"
-                    defaultMessage="Repositories"
-                  />
-                </EuiLink>
-              ),
-            }}
-          />
-        </EuiCallOut>
-        <EuiSpacer />
-      </Fragment>
-    ) : null;
-
     content = (
-      <Fragment>
-        {repositoryErrorsWarning}
-
+      <>
+        {snapshots.length === SNAPSHOT_LIST_MAX_SIZE && (
+          <>
+            <EuiCallOut
+              color="warning"
+              iconType="help"
+              data-test-subj="maxSnapshotsWarning"
+              title={i18n.translate(
+                'xpack.snapshotRestore.snapshotsList.maxSnapshotsDisplayedTitle',
+                { defaultMessage: 'Snapshot list may be incomplete' }
+              )}
+            >
+              <FormattedMessage
+                id="xpack.snapshotRestore.snapshotsList.maxSnapshotsDisplayedDescription"
+                defaultMessage="You have reached the maximum number of snapshots that can be displayed. To manage all of your snapshots, use {docLink}."
```

Comment:
```suggestion
                defaultMessage="You've reached the maximum number of snapshots. To view all of your snapshots, use {docLink}."
```

Reply:
I think that we can shorten this a bit if the message description indicates that we can't show the full list of snapshots.  I also think that saying "To view all of your snapshots" is more accurate, since users can't manage snapshots with the GET snapshot API.